packages/riverpod_lint_flutter_test/test/goldens/lints/avoid_passing_build_context_to_providers.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/widgets.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'avoid_passing_build_context_to_providers.g.dart';
+
+@riverpod
+int fn(
+  FnRef ref,
+  // expect_lint: avoid_passing_build_context_to_providers
+  BuildContext context1, {
+  // expect_lint: avoid_passing_build_context_to_providers
+  required BuildContext context2,
+}) =>
+    0;
+
+@riverpod
+class MyNotifier extends _$MyNotifier {
+  int build() => 0;
```

Comment:
Could you test parameters passed to the build method too?

Reply:
Sure, no problem.